src/controller/DonationsController.ts

```diff
@@ -1,15 +1,17 @@
 import { NextFunction, Request, Response } from "express";
-import * as Stripe from "stripe";
```

Comment:
@mojowen I made a few changes to your work in progress to get this up and running--let me know if you wanna run through the changes sometime this week!